communicator.go

```diff
@@ -97,7 +98,7 @@ func (c *Communicator) reset() {
 	c.keepaliveDone = nil
 
 	if c.client != nil {
-		c.client.Close()
+		_ = c.client.Close()
```

Comment:
Why?

Reply:
I have linters that warn all the time that we ignore returned error